specificatie/openapi.yaml

```diff
@@ -0,0 +1,377 @@
+openapi: 3.0.0
+servers:
+  - description: "APILAB testserver"
+    url: https://www.haalcentraal.nl/haalcentraal/api/brp
+info:
+  title: Bevragingen historie ingeschreven personen
+  description: |
+                API voor het ontsluiten van historische gegevens in relatie tot ngeschreven personen en aanverwante gegevens uit de GBA en RNI. Met deze API worden de historische gegevens van ingeschreven personen, hun kinderen, partners en ouders ontsloten.
+
+                Heeft een persoon bijvoorbeeld geen geldige nationaliteit, dan wordt nationaliteit niet geretourneerd. <br> Heeft een persoon een beëindigd partnerschap of huwelijk, dan wordt de partner niet geretourneerd.
```

Comment:
Dit is niet waar. Deze zinnen hebben betrekking op actuele gegevens. De historische API toont voormalige partner of nationaliteit juist wel

Reply:
In de genereervariant wordt beëindigd in SwaggerHub getoond als be�indigd

Reply:
Daar kan ik niet veel aan doen. Dat heeft te maken met de resolver. Dat gaat overigens in andere repositories wel goed, dus ik ben benieuwd hoe dit op github gaat als daar geresolved wordt.

Reply:
Issue aangemaakt om dit te checken #13